test/regressions/index.tsx

```diff
@@ -1,12 +1,11 @@
 import * as React from 'react';
 import * as ReactDOM from 'react-dom/client';
 import { createBrowserRouter, RouterProvider, Outlet, NavLink, useNavigate } from 'react-router';
-import TestViewer from 'test/regressions/TestViewer';
-import { useFakeTimers } from 'sinon';
 import { Globals } from '@react-spring/web';
-import { setupTestLicenseKey } from '../utils/testLicense';
+import TestViewer from './TestViewer';
+import { generateTestLicenseKey, setupTestLicenseKey } from '../utils/testLicense';
 
-setupTestLicenseKey();
+setupTestLicenseKey(generateTestLicenseKey(new Date('2099-01-01')));
```

Comment:
Clock mocking options used before no longer work on Vite.
Creating a license key with a long expiration to avoid license verification issues.